vita3k/renderer/src/gl/draw.cpp

```diff
@@ -131,8 +131,8 @@ void draw(GLState &renderer, GLContext &context, GxmContextState &state, const F
     glBufferData(GL_ELEMENT_ARRAY_BUFFER, index_size * count, nullptr, GL_DYNAMIC_DRAW);
     glBufferData(GL_ELEMENT_ARRAY_BUFFER, index_size * count, indices, GL_DYNAMIC_DRAW);
 
-    const std::uint8_t *indicies_u8 = reinterpret_cast<const std::uint8_t *>(indices);
-    delete indicies_u8;
+    std::uint8_t *indices_u8 = reinterpret_cast<std::uint8_t *>(indices);
+    delete[] indices_u8;
```

Comment:
This should be moved up to command handler.

Reply:
It's also not protected by deferred context.

Reply:
strange, is already like that on first commit 🤔 
![image](https://user-images.githubusercontent.com/5261759/118348858-564e1880-b54d-11eb-9234-c40dc9ecd592.png)

Reply:
I don't understand? All data submitted from command list is only copied, when execute on immediate

Reply:
Oh I see I thought command list can be executed multiple times.

Reply:
Data copy thing was about copying data from pointer at the point of execution, then indexData shouldn't be included there.

Reply:
Never mind about this.